benchmarks/ner_embeddings/project.yml

```diff
@@ -0,0 +1,278 @@
+      - "python -m scripts.cli_util remove span-labeling-datasets"
+
+
+  - name: "download-models"
+    help: "Download spaCy models for their word-embeddings."
+    script:
+      - "python -m spacy download en_core_web_lg"
+      - "python -m spacy download es_core_news_lg"
+      - "python -m spacy download nl_core_news_lg"
+      - "python -m spacy download fi_core_news_lg"
+
+  - name: "init-fasttext"
+    help: "Initialize the FastText vectors."
+    script:
+      - "python -m scripts.cli_util unzip assets/fasttext.en.gz"
+      - "python -m scripts.cli_util unzip assets/fasttext.es.gz"
+      - "python -m scripts.cli_util unzip assets/fasttext.nl.gz"
+      - "python -m spacy init vectors en assets/fasttext.en vectors/fasttext-en"
+      - "python -m spacy init vectors es assets/fasttext.es vectors/fasttext-es"
+      - "python -m spacy init vectors nl assets/fasttext.nl vectors/fasttext-nl"
+
+  - name: "make-tables"
+    help: "Pre-compute token-to-id tables for MultiEmbed."
+    script:
+      - >-
+        python -m scripts.token_map
+        configs/multiembed.cfg
+        ${vars.tables_path}/${vars.dataset}
+        --min-freq ${vars.min_freq}
+        --paths.train corpus/${vars.dataset}-train.spacy
+        --paths.dev corpus/${vars.dataset}-dev.spacy
+        --code scripts/custom_components.py
+        --paths.tables ${vars.tables_path}
+
+  - name: "init-labels"
+    help: "Initialize labels first before training"
+    script:
+      - >-
+        python -m spacy init labels
+        configs/${vars.ner_config}.cfg
+        corpus/labels/${vars.dataset}.labels
+        --verbose 
+        --paths.train corpus/${vars.dataset}-train.spacy
+        --paths.dev corpus/${vars.dataset}-dev.spacy
+        --paths.tables tables/${vars.dataset}/${vars.dataset}-train.tables
+        --gpu-id ${vars.gpu_id}
+        --code scripts/custom_components.py
+
+  - name: "train"
+    help: "Train NER model."
+    script:
+      - >-
+        python -m spacy train
+        configs/${vars.ner_config}.cfg
+        --nlp.lang ${vars.language}
+        --output training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/
+        --paths.train corpus/${vars.dataset}-train.spacy
+        --paths.dev corpus/${vars.dataset}-dev.spacy
+        --paths.vectors ${vars.vectors}
+        --paths.tables ${vars.tables_path}/${vars.dataset}/${vars.dataset}-train.tables
+        --components.tok2vec.model.embed.include_static_vectors ${vars.include_static_vectors}
+        --nlp.batch_size ${vars.batch_size}
+        --system.seed ${vars.seed}
+        --gpu-id ${vars.gpu_id}
+        --code scripts/custom_components.py
+    deps:
+      - corpus/${vars.dataset}-train.spacy
+      - corpus/${vars.dataset}-dev.spacy
+    outputs:
+      - training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/model-last
+      - training/${vars.dataset}/${vars.vectors}/${vars.ner_config}/${vars.seed}/model-best
+
+  - name: "train-adjust-rows"
```

Comment:
This doesn't actually run for me:
```
=========================== Initializing pipeline ===========================
✘ Config validation error
tok2vec.model.embed -> rows     value is not a valid list
{'@architectures': 'spacy.MultiHashEmbed.v2', 'width': 256, 'attrs': ['NORM', 'PREFIX', 'SUFFIX', 'SHAPE'], 'rows': "'[5000, 2500, 2500, 2500]'", 'include_static_vectors': True}

```

Reply:
Oh no :(. Do you have any hunch why does it work for me and @ljvmiranda921 and not you?

Reply:
I pushed it without the quotes and that also does work for me? Does it work for you now?

Reply:
Such jenk, yeah okay I will add this to the README, so sad.

Reply:
oh the thing is that the `run_experiments.py` wants to run this command as well for one of the experiments.

Reply:
The `_make_train_command` function in `run_experiments.py` generates these `train-adjust-rows` commands for that one particular experiment. We can remove support from reducing this single experiment?

Reply:
Sorry I'm not clear on what you're suggesting to do exactly?

Reply:
I'm a little on the fence, to be honest. I typically want our example projects to also run on Windows, but this is a known bug that we need to look into as part of `weasel` and I don't want to have it blocking this project.

I'll leave the decision to you: either we leave it in and make a note/comment somewhere that this might not work on Windows, or we take it out and explain somewhere how to run those experiments.

Reply:
I would keep it in and explain in the docs that this is a known issue.

Reply:
Wrote some explanation.